src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -1961,6 +1989,17 @@ theorem neg_ne_iff_ne_neg {x y : BitVec w} : -x ≠ y ↔ x ≠ -y := by
     subst h'
     simp at h
 
+/-! ### abs -/
+
+@[simp]
```

Comment:
If you agree that the pattern makes sense, why not `bv_toNat` this one as well?^^

Reply:
Right. I forgot to push this last change. Its in now.